src/sunsynk/definitions/single_phase_16kw.py

```diff
@@ -0,0 +1,40 @@
+"""Sunsynk / Deye 16kW hybrid inverter sensor definitions."""
+
+from sunsynk import AMPS, CELSIUS, KWH, VOLT, WATT
+from sunsynk.definitions import COMMON, PROG_CHARGE_OPTIONS, PROG_MODE_OPTIONS
```

Comment:
You need Ll the single phase ones as base

```suggestion
from sunsynk.definitions.single_phase import SENSORS as COMMON
```